securedrop/source.py

```diff
@@ -124,6 +124,19 @@ def generate_unique_codename(num_words):
     """Generate random codenames until we get an unused one"""
     while True:
         codename = crypto_util.genrandomid(num_words)
+
+        # THe maximum length of a word in the wordlist is 6, and the maximum
+        # codename length is 10, so it is currently impossible to generate a
```

Comment:
The numbers 6 and 10 refer to different units (characters and words, respectively); add the units for clarity.